files/jsondata/L10n-Common.json

```
@@ -35,6 +37,7 @@
     "es": "Guías",
     "fr": "Guides",
     "ja": "ガイド",
+    "ko": "가이드",
```

```suggestion
    "ko": "안내서",
```

https://github.com/mdn/translated-content/blob/main/docs/ko/guides/glossary-guide.md#section-subtitle 에 따라 안내서로 변경 부탁드립니다! (참고: https://github.com/mdn/translated-content/issues/11093#issuecomment-1411005106)